Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.py

```diff
@@ -0,0 +1,880 @@
+        show_actual_incident, incident_id, include_indicators_similarity
+
+
+def load_current_incident(incident_id: str, populate_fields: List[str]):
+    """
+    Load current incident if incident_id given or load current incident investigated
+    :param incident_id: incident_id
+    :param populate_fields: populate_fields
+    :return:
+    """
+    if not incident_id:
+        incident = demisto.incidents()[0]
+        cf = incident.pop('CustomFields', {}) or {}
+        incident.update(cf)
+        incident = {k: v for k, v in incident.items() if k in populate_fields}
+        incident_id = incident['id']
+    else:
+        incident = get_incident_by_id(incident_id, populate_fields)
+        if not incident:
+            return None, incident_id
+    return incident, incident_id
+
+
+def remove_fields_not_in_incident(*args, incorrect_fields):
+    """
+    Return list without field in incorrect_fields
+    :param args: *List of fields
+    :param incorrect_fields: fields that we don't want
+    :return:
+    """
+    return [[x for x in field_type if x not in incorrect_fields] for field_type in args]
+
+
+def get_similar_incidents_by_indicators(args: Dict):
+    """
+    Use DBotFindSimilarIncidentsByIndicators automation and return similars incident from the automation
+    :param args: argument for DBotFindSimilarIncidentsByIndicators automation
+    :return:  return similars incident from the automation
+    """
+    res = demisto.executeCommand('DBotFindSimilarIncidentsByIndicators', args)
+    if is_error(res):
+        return_error(get_error(res))
+    return res[1]['Contents']
+
+
+def dumps_json_field_in_incident(incident: Dict):
+    """
+    Dumps value that are dict in for incident values
+    :param incident: json representing the incident
+    :return:
+    """
+    for field in incident.keys():
+        if isinstance(incident[field], dict):
+            incident[field] = json.dumps(incident[field])
+    incident_df = pd.DataFrame.from_dict(incident, orient='index').T
+    return incident_df
+
+
+def return_outputs_summary(confidence: float, number_incident_fetched: int, number_incidents_found: int,
+                           fields_used: List[str], global_msg: str) -> None:
+    """
+    Return entry for summary of the automation - Give information about the automation run
+    :param confidence: confidence level given by the user
+    :param number_incident_fetched: number of incident fetched from the instance
+    :param number_incidents_found: number of similar incident found
+    :param fields_used: Fields used to find similarity
+    :param global_msg: informative message
+    :return:
+    """
+    summary = {
+        'Confidence': str(confidence),
+        'Incident fetched with exact match': number_incident_fetched,
+        'Number of similar incident found ': number_incidents_found,
+        'Valid fields used for similarity': ', '.join(fields_used),
+        'Additional message': global_msg
```

Comment:
1. I'd extract the `Additional message` outside the table, and just print it above/beneath the table as a text
2. If additional message is empty, no need to display it